packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
+          decrementValue={this.decrementValue}
+          formatValue={this.formatValue}
+          inputClassName={getClassName('bpk-nudger-configurable')}
+        />
+      </div>
+    );
+  }
+}
+```
+
 ## Props
 
-| Property              | PropType                      | Required | Default Value |
-| --------------------- | ----------------------------- | -------- | ------------- |
-| id                    | string                        | true     | -             |
-| decreaseButtonLabel   | string                        | true     | -             |
-| increaseButtonLabel   | string                        | true     | -             |
-| max                   | number                        | true     | -             |
-| min                   | number                        | true     | -             |
-| onChange              | func                          | true     | -             |
-| value                 | number                        | true     | -             |
-| className             | string                        | false    | null          |
-| buttonType            | oneOf('secondary', 'outline') | false    | secondary     |
+### BpkNudger
+
+| Property            | PropType                      | Required | Default Value |
+| ------------------- | ----------------------------- | -------- | ------------- |
+| id                  | string                        | true     | -             |
+| decreaseButtonLabel | string                        | true     | -             |
+| increaseButtonLabel | string                        | true     | -             |
+| max                 | number                        | true     | -             |
+| min                 | number                        | true     | -             |
+| onChange            | func                          | true     | -             |
+| value               | number                        | true     | -             |
+| className           | string                        | false    | null          |
+| buttonType          | oneOf('secondary', 'outline') | false    | secondary     |
+
+### BpkConfigurableNudger
+
+| Property            | PropType                      | Required | Default Value |
+| ------------------- | ----------------------------- | -------- | ------------- |
+| id                  | string                        | true     | -             |
+| decreaseButtonLabel | string                        | true     | -             |
+| increaseButtonLabel | string                        | true     | -             |
+| max                 | number                        | true     | -             |
+| min                 | number                        | true     | -             |
+| onChange            | func                          | true     | -             |
+| value               | number                        | true     | -             |
+| compareValues       | func                          | true     | -             |
```

Comment:
We need to better document these functions. Especially `compareValues`

Reply:
Done